### ISSUE TITLE
[Chains] add geth as a viable chain type && expose commands for setup needs

### ISSUE THREAD

**Issue by zramsay** [eris-ltd/eris#692]
- a `services exec` wrapper for doing things with eth chains through geth.
- to occur in parallel with pulling out start script from geth image see (https://github.com/eris-ltd/common/issues/40) and tut for using eris and eth (see https://github.com/eris-ltd/docs.erisindustries.com/issues/99)
- todo: figure out where https://github.com/eris-ltd/geth-tmsp fits in (as an alternative option, presumably)
## Actions
- create account
- start single (private) node
- start full node & connect to public eth chain
- the above w/ mining
- deploy contracts to either any of these chains
- easily send tx's (we should do the same with BTC, where the priv_key can be imported into container)

The above actions will leverage both the [eth-client](https://github.com/eris-ltd/eth-client), geth, and the compilers to get the job done.

**Comment by zramsay** [comment]
closing as low priority & not current direction of platform

**Comment by VoR0220** [comment]
@zramsay @compleatang consider reopening this. I just got an idea for an integration test that we could hypothetically run against both EDB and Geth to ensure that we are getting better throughput/performance in our chain vs eth's chain (just by running EPM tests against it). Not only that, but there will be demand to have our tools available for use in the public blockchain setting. If nothing else, we will have something to assert why our product ought to be used.
